i18n/checkLocaleKeys.js

```diff
@@ -21,11 +22,18 @@ const checkLocaleKeys = () => {
 
     const referenceKeys = extractAllKeys(enFileContent)
 
-    //check if in CI/CD
+    //check if in CI/CD and auto update and push
     if (process.env.NODE_ENV === 'production') {
-        runCICDChecksForLocaleFiles(localeFilesWithoutEnFileName, referenceKeys)
+        const missingKeysSummary = runCICDChecksForLocaleFiles(localeFilesWithoutEnFileName, referenceKeys)
+        if (missingKeysSummary.length) {
+            findAndInsertMissingKeysAndValuesInNonEnFiles(
+                localeFilesWithoutEnFileName, enFileContent, referenceKeys, insertMissingKeysAndValues
+            )
+            // Commit and push changes if any files were updated
+            commitAndPushChangesForLocaleFiles()
```

Comment:
I don't think we should push in the CI command. This could cause errors that are missed and it also can make people unaware of what's actually happening (since it's only revealed in the logs). I think we need some way to make this visible in the PR like a comment  (see the socket.io results as an example)

Reply:
I think this is a solid functiuon that could be triggered from the github comment though

Reply:
I will see what I can do!